src/common/styles-dictionary/scss/_variables.scss

```diff
@@ -0,0 +1,226 @@
+
+// Do not edit directly
+// Generated on Mon, 15 Nov 2021 22:54:50 GMT
+
+$sds-font-body-xxxs-400-font-family: 'Open Sans', 'OpenSans-Regular';
+$sds-font-body-xxxs-400-font-size: 11px;
+$sds-font-body-xxxs-400-letter-spacing: 0.3px;
+$sds-font-body-xxxs-400-line-height: 16px;
+$sds-font-body-xxxs-400-font-weight: 400;
+$sds-font-body-xxxs-400-color: black;
+$sds-font-body-xxxs-600-font-family: 'Open Sans', 'OpenSans-SemiBold';
+$sds-font-body-xxxs-600-font-size: 11px;
+$sds-font-body-xxxs-600-letter-spacing: 0.3px;
+$sds-font-body-xxxs-600-line-height: 16px;
+$sds-font-body-xxxs-600-font-weight: 600;
+$sds-font-body-xxxs-600-color: black;
+$sds-font-body-xxxs-button-font-family: 'Open Sans', 'OpenSans-Regular';
```

Comment:
I'm wondering what this button-specific font is since the fonts on our buttons should tie back to existing text styles. Is there something we need to update on our end?

Reply:
nope, you're correct. it's the same font. These values were taken from the defaultTheme.ts file. Looking into it, I don't see the button font being used anywhere. I'll do a quick check with the clients before removing it

Reply:
it's part of the MUI default theme we created for czifui so I'm removing it from the styles-dictionary but leaving it in defaultTheme.ts